app/views/org_admin/sections/_edit.html.erb

```diff
@@ -23,46 +27,46 @@
       <div class="form-group col-md-10">
         <%= f.label(:description, _('Description'), class: "control-label") %>
         <div data-toggle="tooltip" title="<%= _("Enter a basic description. This could be a summary of what is covered in the section or instructions on how to answer. This text will be displayed in the coloured banner once a section is opened to edit.") %>">
-          <%= text_area_tag('section-desc', section.description, class: "section tinymce form-control") %>
+          <%= f.text_area(:description, class: "section tinymce form-control") %>
```

Comment:
thank you. I hated that these were outside the 'model[attribute]' pattern before and therefore not able to easily get them into the protected params section of the controllers